### PR TITLE
[extension-selenium] Log elements filtered by visibility

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/AbstractElementAction.java [hunk 1]
```diff
@@ -105,7 +108,13 @@ public LocatorType getType()
                 LOGGER.warn(e.getMessage(), e);
                 return false;
             }
-        }).collect(Collectors.toList());
+        }).collect(Collectors.collectingAndThen(Collectors.toList(), list ->
+        {
+            LOGGER.atInfo().addArgument(visibility)
+                           .addArgument(list::size)
+                           .log("Number of {} elements is equal to {}");
```
Comment (valfirst) [inline vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/AbstractElementAction.java:115]: how does the message look like?

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/AbstractElementAction.java [hunk 2]
```diff
@@ -70,15 +70,17 @@ public LocatorType getType()
                 ? waitForElement(searchContext, locator)
                 : searchContext.findElements(locator);
         boolean elementsFound = null != elements;
-        LOGGER.info("Total number of elements found {} is equal to {}", locator, elementsFound ? elements.size() : 0);
+        LOGGER.atInfo().addArgument(locator)
+                       .addArgument(() -> elementsFound ? elements.size() : 0)
+                       .log("Total number of elements found {} is equal to {}");
```
Comment (valfirst) [inline vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/AbstractElementAction.java:75]:
```
"Total number of elements found {} is {}"
```
?

File: vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/AbstractElementAction.java (hunk 1 above)
Comment (valfirst) [inline vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/AbstractElementAction.java:115]:
```
"Number of {} elements is {}"
```

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/AbstractElementAction.java (hunk 1 above)
Comment (uarlouski) [reply vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/AbstractElementAction.java:115]: Number of VISIBLE elements is equal to 1

File: vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/AbstractElementAction.java (hunk 2 above)
Comment (uarlouski) [reply vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/AbstractElementAction.java:75]: what if someone uses this message in known issue?

**Review by ikalinin1 — APPROVED** [review]

File: vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/AbstractElementAction.java (hunk 1 above)
Comment (valfirst) [reply vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/AbstractElementAction.java]: it should be like this: `Number of visible elements is 1`

**Review by valfirst — APPROVED** [review]

**Pull request merged**